docs/services/sharelatex.md

```diff
@@ -63,23 +63,34 @@ ShareLaTeX requires a redis and a mongodb instance!
 ```
 
 ### Installation of texlive-full
-!!! warning ""
+**Attention**
     If you start the container using docker-compose, the image will be commited with all environment variables and labels.
 
-1. Create admin user
-```
-docker-compose exec paper /bin/bash -c "cd /var/www/sharelatex; grunt user:create-admin --email my@email.address"
-```
+1. Install `texlive-full`
 
-2. Install `texlive-full`
-```
-# you should do this in a screen, cause it will take a couple of hours...
-screen -AmdS latex-installation 'docker-compose exec paper tlmgr install scheme-full'
-```
+   **Note:**
+
+   - You should do this in a screen, cause it will take a couple of hours...
+   - The image will be about 8 gigabytes after installing all the additional packages.
```

Comment:
This is not a list, and you didn't end the line with two spaces, so these two lines will probably look like this.
```
 - You should do this in a screen, cause it will take a couple of hours...- The image will be about 8 gigabytes after installing all the additional packages.
 ```